src/sidebars/star/star.html

```diff
@@ -2,7 +2,8 @@
      ng-mouseenter="starCtrl.mouseEnter()" ng-mouseleave="starCtrl.mouseLeave()" title="{{starCtrl.tooltip()}}" draggable="false" />
 <div class="confirmation-backdrop" ng-if="starCtrl.confirmationShow" ng-mousedown="starCtrl.hideModal(); $event.stopPropagation();">
   <div class="bubble-head" ng-style="{top:starCtrl.modalBubbleTop()+'px'}"></div>
-  <div class="confirmation-modal" ng-class="{flipped: starCtrl.modalFlip()}"  ng-click="$event.stopPropagation();" ng-style="{top:starCtrl.modalTop()+'px'}">
+  <div class="confirmation-modal" ng-class="{flipped: starCtrl.modalFlip()}" ng-click="$event.stopPropagation();"
```

Comment:
Actually, it is needed. You can click through it and select another stock on non-compact mode (same with double click, and I'll push that fix up in a sec)